.gitignore

```diff
@@ -6,3 +6,4 @@
 # Mac OS System
 .DS_Store*
 ._*
+/bin/
```

Comment:
Can you please not modify .gitignore in this PR. Just keep focusing on the original purpuse of your PR 😉  Thanks!